### PR TITLE
[SPARK-32234][FOLLOWUP][SQL]Update the description of utility method

### PR DESCRIPTION
**Pull request by SaurabhChawla100**
### What changes were proposed in this pull request?
As the part of this PR https://github.com/apache/spark/pull/29045 added the helper method. This PR is the FOLLOWUP PR to update the description of helper method.

### Why are the changes needed?
For better readability and understanding of the code 

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Since its only change of updating the description , So ran the Spark shell

### CODE REVIEW

**Comment by SaurabhChawla100** [comment]
cc @maropu @cloud-fan @dongjoon-hyun @gatorsmile

**Comment by dongjoon-hyun** [comment]
ok to test

File: sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala [hunk 1]
```diff
@@ -207,10 +207,19 @@ object OrcUtils extends Logging {
...
   /**
-   * @return Returns the result schema string based on the canPruneCols flag.
-   *         resultSchemaString will be created using resultsSchema in case of
-   *         canPruneCols is true and for canPruneCols as false value
-   *         resultSchemaString will be created using the actual dataSchema.
+   * This method returns the result schema as string based on the canPruneCols flag.
+   * resultSchemaString will be created using pruned col in case of
+   * canPruneCols is true and for canPruneCols as false value
+   * resultSchemaString will be created using the actual dataSchema.
+   * Also as the part of this method update the MAPRED_INPUT_SCHEMA
```
Comment (cloud-fan) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala:214]: `update the MAPRED_INPUT_SCHEMA` -> `updates the MAPRED_INPUT_SCHEMA in the given 'conf'`.

**Review by cloud-fan — APPROVED** [review]

File: sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala [hunk 2]
```diff
@@ -207,10 +207,19 @@ object OrcUtils extends Logging {
   }
 
   /**
-   * @return Returns the result schema string based on the canPruneCols flag.
-   *         resultSchemaString will be created using resultsSchema in case of
-   *         canPruneCols is true and for canPruneCols as false value
-   *         resultSchemaString will be created using the actual dataSchema.
+   * This method returns the result schema as string based on the canPruneCols flag.
```
Comment (HyukjinKwon) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala:210]: I would just write as:

Returns the result schema to read from ORC file. In addition, It sets the schema string to 'orc.mapred.input.schema' so ORC readers can use later.

and inline the details about parameters into the parameter descriptions below.

Comment (SaurabhChawla100) [reply sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala]: that looks fine. Will update as suggested

File: sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala (hunk 1 above)
Comment (SaurabhChawla100) [reply sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcUtils.scala]: Updated the comment as suggested by @HyukjinKwon

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
retest this please

**Comment by cloud-fan** [comment]
github action passes, merging to master, thanks!

**Pull request closed without merging**